src/store.rs

```diff
@@ -22,7 +22,7 @@ pub struct SavedOutput {
 	pub rate: f32,
 	pub x: i32,
 	pub y: i32,
-	//pub rotation: Rotation,
+	pub transform: String,
```

Comment:
This could be an enum

Reply:
I'm not sure I follow here. This may be my general inexperience talking but since we're parsing the same string we want to output, what is the benefit of transforming this to an enum? Could you briefly explain how you would see this?

Reply:
This ensures the value is valid, and isn't garbage

Reply:
Isn't this redundant with the matching in the frontend? 

The way I've got it right now (not committed) is that it matches the string against the valid options and passes through if matched or passes "transform normal" if not. This serves to both handle garbage and reset the output to the normal rotation if transform is absent (something swaymsg doesn't do on it's own if the output already has a rotation prior to passing a new command).

If you'd rather I change this to an enum I'll gladly do so, but then there's little point in matching in the frontend as the parser will silently fail before it gets there in the case of incorrect input.

Reply:
>Isn't this redundant with the matching in the frontend?

All frontends and backends may not use the same string.

> but then there's little point in matching in the frontend as the parser will silently fail before it gets there in the case of incorrect input.

The parser silently failing is a separate issue that needs to be fixed anyway. See https://github.com/emersion/kanshi/issues/18